### PR TITLE
Add picomodal@3.0.0 w/ git autoupdate

### LINKED ISSUE #10282: [Request] Add picomodal
**Library name:** picomodal
**Git repository url:** https://github.com/Nycto/PicoModal
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #10282 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Nycto/PicoModal
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/picomodal
 * License and its reference: MIT, https://github.com/Nycto/PicoModal#license
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 163
   - Count of forks: 32
 * NPM download stats (optional):
   - Downloads in the last day: 70
   - Downloads in the last week: 347
   - Downloads in the last month: 1563

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #10282, cc @Nycto

### CODE REVIEW

**Comment by qwe19272375** [comment]
@pvnr0082t Please help me review this pull request, thanks!

File: ajax/libs/picomodal/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Nycto/PicoModal.git"
+  },
+  "keywords": [
+    "modal",
+    "popup",
+    "vanilla"
+  ],
+  "npmName": "picomodal",
```
Comment (pvnr0082t) [inline ajax/libs/picomodal/package.json]: @qwe19272375 There are 5 releases on NPM but 9 on GitHub. Let's use git auto-update config instead.

**Comment by qwe19272375** [comment]
`.map` file added and using git auto-update now. 😃 
@pvnr0082t Please help me review this PR again, thanks!

**Comment by pvnr0082t** [comment]
ping @cdnjs/intern2

**Comment by maruilian11** [comment]
I think this PR is fine. I have helped rebase with the latest master branch.

**Comment by qwe19272375** [comment]
Thanks

**Pull request merged**